lib/dish.rb

```diff
@@ -0,0 +1,9 @@
+class Dish
```

Comment:
This currently doesn't seem to be being used by the Menu class, and currently is pretty much equivalent to just being a hash, eg `{ name: "dal",  price:"4.00" }`. 

A Dish class could be useful down the line, if you wanted to allow customisation (Dal without the lentils for me, thanks), but for now I don't think it's necessary.